### PR TITLE
fix: Replace discovery protocol tokio interval

### PR DESCRIPTION
**Pull request by TheWaWaR**

### CODE REVIEW

**Comment by TheWaWaR** [comment]
tests passed

File: tests/test_block_send_session.rs [hunk 1]
```diff
@@ -0,0 +1,159 @@
+use bytes::Bytes;
```
Comment (quake) [inline tests/test_block_send_session.rs]: this test looks like not related to discovery interval issue, could you please move it to other PR?

Comment (TheWaWaR) [reply tests/test_block_send_session.rs]: OK, I'll issue a PR fix those tests first
